### ISSUE TITLE
Type information loading indefinitely.

### ISSUE THREAD

**Issue by lelabo-m** [mylesmmurphy/prettify-ts#16]
Type: <b>Bug</b>

Vscode 1.87

I don't really know why it started to take more time and then stop working at once indefinitely... I just know that bissecting the extensions, it seems to be the culprit.

Extension version: 0.0.17
VS Code version: Code 1.87.0 (019f4d1419fbc8219a181fab7892ebccf7ee29a2, 2024-02-27T23:41:44.469Z)
OS version: Windows_NT x64 10.0.26063
Modes:
Connection to 'wsl+Ubuntu-22.04' could not be established

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz (8 x 2112)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|15.92GB (3.03GB free)|
|Process Argv|--folder-uri=vscode-remote://wsl+Ubuntu-22.04/home/lelabo/DEV/SNACS/willitfodmap --remote=wsl+Ubuntu-22.04 --crash-reporter-id 5a6117dd-b4ec-43be-a7cd-d45c57b974b8|
|Screen Reader|no|
|VM|0%|

Connection to 'wsl+Ubuntu-22.04' could not be established
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805cf:30301675
binariesv615:30325510
vsaa593cf:30376535
py29gd2263:30899288
c4g48928:30535728
azure-dev_surveyonecf:30548226
a9j8j154:30646983
962ge761:30959799
pythongtdpath:30769146
welcomedialog:30910333
pythonidxpt:30866567
pythonnoceb:30805159
asynctok:30898717
pythontestfixt:30902429
pythonregdiag2:30936856
pyreplss1:30897532
pythonmypyd1:30879173
pythoncet0:30885854
pythontbext0:30879054
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
cppperfcontrol:30979541
d34g3935:30971562
fegfb526:30981948
bg6jg535:30979843
78248357:30976999
dsvsc020:30976470

```

</details>

**Comment by mylesmmurphy** [comment]
Hey @lelabo-m! Thanks for bringing this to my attention. Would you mind sharing more details about when you run into issues? Are you able to share which type is loading forever?

**Comment by mylesmmurphy** [comment]
Hey @lelabo-m !
Prettify TS is currently being completely re-engineered, which should include massive performance increases. If interested in testing out, simply download the vsix file under the release and install using the official VSCode instructions found [here](https://code.visualstudio.com/docs/editor/extension-marketplace#_install-from-a-vsix). If you try it out, please let me know if your performance issues are resolved with this!

https://github.com/mylesmmurphy/prettify-ts/releases/tag/v0.1.0-alpha

**Comment by mylesmmurphy** [comment]
Hi @lelabo-m 

I just released v0.1.5 of the extension which includes a massive overhaul of how it behaves under the hood, as well as many customization options in the settings. Please try this version and let me know if your problems are resolved.

**Comment by mylesmmurphy** [comment]
Per issue #21 
Performance issues should be resolved, please open a new issue if they continue to occur
